src/engine/server/databases/mysql.cpp

```diff
@@ -71,6 +71,7 @@ class CMysqlConnection : public IDbConnection
 	void Print(IConsole *pConsole, const char *pMode) override;
 
 	const char *BinaryCollate() const override { return "utf8mb4_bin"; }
+	const char *Decimal() const override { return "DECIMAL(65, 1)"; }
```

Comment:
```suggestion
	const char *Decimal() const override { return "DECIMAL(65, 1)"; }
```
I think we have two digits past the decimal point.

How does this affect the size of the database? Do we need to reduce the `65` in `DECIMAL(65, 1)`?